### DISCUSSION TITLE
Path Exclude

### DISCUSSION THREAD

**Discussion by alabotski** [d4rken-org/sdmaid-se#898] [Q&A]
Hi, I have some questions 

1. Why does scanning take so long? Why can’t I immediately select a path or enter it manually?
[Screenshot_20231227-112430](https://github.com/d4rken-org/sdmaid-se/assets/439337/684a6c25-e694-4a67-add4-2bbc7455ad04)

2. For some reason, there is no folder /storage/emulated/0/Android
[Screenshot_20231227-114846](https://github.com/d4rken-org/sdmaid-se/assets/439337/3ae30ae8-7ec8-40be-a93e-6bd88a5c8abe)

3. But I can import this folder
```
{
  "exclusionRaw":"[{\"path\":{\"file\":\"/storage/emulated/0/Android\",\"pathType\":\"LOCAL\",\"segmentsCache\":[\"\",\"storage\",\"emulated\",\"0\",\"Android\"]},\"tags\":[\"SYSTEMCLEANER\"]}]",
  "version":1
}
```
[Screenshot_20231227-114737](https://github.com/d4rken-org/sdmaid-se/assets/439337/4132ee3e-7097-4bf5-9e4b-5f4c690d5ad5)

**Comment by d4rken** [comment]
Please use [discussions](https://github.com/d4rken-org/sdmaid-se/discussions) for just questions, keep [issues](https://github.com/d4rken-org/sdmaid-se/issues) for "actionable" things.

> 1. Why does scanning take so long? Why can’t I immediately select a path or enter it manually?
>    [Screenshot_20231227-112430](https://github.com/d4rken-org/sdmaid-se/assets/439337/684a6c25-e694-4a67-add4-2bbc7455ad04)

The analyzer tool is not a "file explorer". To be able to show how large something is and how it compares to other things, all paths on a storage device have to be scanned.

> 2. For some reason, there is no folder /storage/emulated/0/Android
>    [Screenshot_20231227-114846](https://github.com/d4rken-org/sdmaid-se/assets/439337/3ae30ae8-7ec8-40be-a93e-6bd88a5c8abe)

The screenshot shows "user files"/"user data". `Android/` is not where users put their files, but apps put their files. It is reserved for public app data (if devs did their job right). 
To browse that you'd view the app list in storage analyzer.

The idea here being: You don't free space by randomly deleting folders in `Android/`, but by finding apps that take up large amounts of space and dealing with them by uninstalling or cleaning (note to self, need to shortcut the AppCleaner here somehow...). The storage analyzer shows space occupied on a specific storage device, divided into user files, apps and system.
